docs/eventing/samples/gcp-pubsub-source/trigger.yaml

```diff
@@ -10,9 +10,7 @@ spec:
       revisionTemplate:
         spec:
           container:
-            # This corresponds to
-            # https://github.com/knative/eventing-sources/blob/release-0.3/cmd/message_dumper/dumper.go
-            image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/message_dumper@sha256:8423232db7a7b4010c0cfbfaef95745efe962631af9b7456903825801a7893f7
+            image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display@sha256:8423232db7a7b4010c0cfbfaef95745efe962631af9b7456903825801a7893f7
```

Comment:
SHA definitely needs to change. Probably wait until 0.5.0 has been cut, then use that. I recommend leaving a comment describing where that image comes from (e.g. 'This corresponds to https://github.com/knative/eventing-sources/blob/v0.5.0/cmd/event_display/main.go.')